py/vtdb/db_object_range_sharded.py

```diff
@@ -400,28 +422,29 @@ def update_sharding_key_entity_id_lookup(class_, cursor_method,
   @db_object.write_db_class_method
   def insert_primary(class_, cursor, **bind_vars):
     if class_.columns_list is None:
-      raise dbexceptions.ProgrammingError("DB class should define columns_list")
+      raise dbexceptions.ProgrammingError('DB class should define columns_list')
 
     query, bind_vars = class_.create_insert_query(**bind_vars)
     cursor.execute(query, bind_vars)
     return cursor.lastrowid
 
-
   @classmethod
   def insert(class_, cursor_method, **bind_vars):
-    """ This method creates the lookup relationship as well as the insert
-    in the primary table. The creation of the lookup entry also creates the
-    primary key for the row in the primary table.
-
-    The lookup relationship is determined by class_.column_lookup_name_map and the bind
-    variables passed in. There are two types of entities -
-    1. Table for which the entity that is also the primary sharding key for this keyspace.
-    2. Entity table that creates a new entity and needs to create a lookup between
-    that entity and sharding key.
+    """Creates the lookup relationship and inserts in the primary table.
+
+     The creation of the lookup entry also creates the primary key for
+    the row in the primary table.
+
+    The lookup relationship is determined by class_.column_lookup_name_map
+    and the bind variables passed in. There are two types of entities -
```

Comment:
Please use a colon at the end here as well.

    ... types of entities:

    1. Table